frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx

```diff
@@ -51,22 +55,55 @@ export const EmbedHomepage = () => {
   }, [plan]);
 
   const onDismiss = (reason: EmbedHomepageDismissReason) => {
-    dispatch(updateSetting({ key: "embedding-homepage", value: reason }));
+    if (reason === "dismissed-run-into-issues") {
+      setFeedbackModalOpened(true);
+    } else {
+      dispatch(updateSetting({ key: "embedding-homepage", value: reason }));
+    }
+  };
+
+  const onFeedbackSubmit = ({
+    feedback,
+    email,
+  }: {
+    feedback?: string;
+    email?: string;
+  }) => {
+    dispatch(
+      updateSetting({
+        key: "embedding-homepage",
+        value: "dismiss-run-into-issues",
+      }),
+    );
+
+    setFeedbackModalOpened(false);
+    sendProductFeedback({
+      comments: feedback,
```

Comment:
I see 2 issues here.
1. I don't know if we want to differentiate between `comment` and `feedback` could they be the same name?
2. `comments` feels like this is an array of comments. Think multiple Facebook, YouTube comments. Where 1 comment could be as long as users want. In this case I think `comment` without an "s" would work better, though point 1) still applies, maybe this could just be named `feedback`.

Reply:
`comments` is what the endpoint on HM expects and to keep things simple the BE only proxies things without changing the name. Do you think it'd be better if it was called `comments` everywhere?

Reply:
In that case I think it's less ambiguous maybe to just name our variable `comment` then.

Reply:
without the `s`?

Reply:
yeah, the point is that `comment` looks closer to `comments` while maintaining the clarity that it's a singular comment. `feedback` is somewhat special and I don't like using it as it's an uncountable noun so `feedbacks` is grammatically incorrect.

Reply:
Makes sense, renamed it in 25e60b3ecdb661645d66295a176cd9053601bcdb